script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -97,9 +97,7 @@ Arcueid abruptly stops in her tracks and glares over her shoulder at me.
 [sha:79c9c98734833ee73e02182fd8051564d1315cbf]{
 -- Page 280, Offset 17257.
 -- 「ついてこないで。あなたみたいな普通の人間にいられると迷惑だって、分からない？」
-"Stop following me.
- An ordinary human such as yourself will only get in my way,
- can't you understand that?"
+"Stop following me. Don't you know that an ordinary human like you would just get in my way?"
```

Comment:
Prefer original here, mainly just due to matching the sentence structure (分らない at the end) and since I think 分る does trend slightly more to _understand_ than _know_, which are subtly different